scripts/agd-builder.sh

```diff
@@ -194,7 +197,8 @@ $do_not_build || (
     test -z "$src" || {
       echo "At least $src is newer than node_modules"
       rm -f "$STAMPS/yarn-built"
-      lazy_yarn install
+      # Ignore engines since we already checked officially supported versions above
```

Comment:
consider prompting to solve the `engines` info
```suggestion
      # Ignore engines since we already checked officially supported versions above
      # UNTIL https://github.com/Agoric/agoric-sdk/issues/9622
```